### PR TITLE
Added the skipFirst option for warmup delays

### PR DESCRIPTION
**Pull request by sujitdmello**
Command-line switch added to allow the first X requests to be discarded from the results calculations:
```
--skipFirst=XX
```

Example output:
image

### CODE REVIEW

File: cmd/ghz/main.go [hunk 1]
```diff
@@ -99,7 +103,7 @@ var (
 		Short('x').Default("0").IsSetByUser(&isXSet).Duration()
 
 	isZStopSet = false
-	zstop      = kingpin.Flag("duration-stop", "Specifies how duration stop is reported. Options are close, wait or ignore.").
+	zstop      = kingpin.Flag("duration-stop", "Specifies how duration stop is reported. Options are close, warnit or ignore.").
```
Comment (michaelperel) [inline cmd/ghz/main.go]: Is this change intended?

File: runner/options.go [hunk 2]
```diff
@@ -458,7 +470,7 @@ func WithProtoFile(proto string, importPaths []string) Option {
 		proto = strings.TrimSpace(proto)
 		if proto != "" {
 			if filepath.Ext(proto) != ".proto" {
-				return errors.Errorf(fmt.Sprintf("proto: must have .proto extension"))
+				return errors.Errorf("proto: must have .proto extension")
```
Comment (michaelperel) [inline runner/options.go]: `errors.Errorf` would be typically used with a format string.

Consider using `errors.New("")` here since there is no format string.

Comment (michaelperel) [inline runner/options.go]: Typically `errors.Errorf` is used with a format string. Consider using `errors.New`, as there is no format string here

File: runner/reporter.go [hunk 3]
```diff
@@ -133,9 +134,17 @@ func newReporter(results chan *callResult, c *RunConfig) *Reporter {
 
 // Run runs the reporter
 func (r *Reporter) Run() {
+	var skipCount uint
+	if r.config.skipFirst > 0 {
+		println("Skipping the first", r.config.skipFirst, "requests from the results calculations.")
```
Comment (michaelperel) [inline runner/reporter.go]: Consider using `fmt.Println` here instead of `println`, as `println` is built in to the runtime and may be removed. See [this stackoverflow post](https://stackoverflow.com/questions/14680255/difference-between-fmt-println-and-println-in-go) for more details

File: cmd/ghz/main.go (hunk 1 above)
Comment (sujitdmello) [reply cmd/ghz/main.go]: No :(

File: runner/options.go (hunk 2 above)
Comment (sujitdmello) [reply runner/options.go]: This was due to a lint error - I just copied the recommended fix. But will do yours.

**Review by michaelperel — APPROVED** [review]
